src/tvdb/client.py

```diff
@@ -97,56 +112,79 @@ def id(self, value: int | str) -> None:  # pyright: ignore[reportPropertyTypeMis
 
     @classmethod
     @abstractmethod
-    async def fetch(cls, media_id: int | str, *, client: "TvdbClient", extended: bool = False) -> "_Media": ...
-
+    def supports_meta(cls, meta: FetchMeta) -> bool:
+        """Check if the class supports a specific meta."""
+        ...
 
-@final
-class Movie(_Media):
-    """Class to interact with the TVDB API for movies."""
-
-    @override
     @classmethod
-    async def fetch(cls, media_id: int | str, client: "TvdbClient", *, extended: bool = False) -> "Movie":
+    async def fetch(
+        cls: type[Self],
+        media_id: int | str,
+        client: "TvdbClient",
+        *,
+        extended: bool = False,
+        short: bool = True,
+        meta: FetchMeta | None = None,
+    ) -> Self:
         """Fetch a movie by its ID.
 
         :param media_id:  The ID of the movie.
         :param client:  The TVDB client to use.
         :param extended:  Whether to fetch extended information.
+        :param short:  Whether to omit characters and artworks from the response. Requires extended=True to work.
+        :param meta:  The meta to fetch. Requires extended=True to work.
         :return:
         """
         media_id = parse_media_id(media_id)
-        response = await client.request("GET", f"movies/{media_id}" + ("/extended" if extended else ""))
-        response = MovieResponse(**response) if not extended else MovieExtendedResponse(**response)  # pyright: ignore[reportCallIssue]
+        query: dict[str, str] = {}
+        if extended:
+            if meta:
+                query["meta"] = meta.value
+            if short:
+                query["short"] = "true"
+            else:
+                query["short"] = "false"
+        elif meta:
+            raise BadCallError("Meta can only be used with extended=True.")
+        response = await client.request(
+            "GET",
+            f"{cls.ENDPOINT}/{media_id}" + ("/extended" if extended else ""),
+            query=query if query else None,
+        )
+        response = cls.ResponseType(**response) if not extended else cls.ExtendedResponseType(**response)  # pyright: ignore[reportCallIssue]
         return cls(client, response.data)
 
 
 @final
-class Series(_Media):
-    """Class to interact with the TVDB API for series."""
+class Movie(_Media):
+    """Class to interact with the TVDB API for movies."""
+
+    ENDPOINT: ClassVar[str] = "movies"
+
+    ResponseType = MoviesIdGetResponse
+    ExtendedResponseType = MoviesIdExtendedGetResponse
 
     @override
     @classmethod
-    async def fetch(cls, media_id: int | str, client: "TvdbClient", *, extended: bool = False) -> "Series":
-        """Fetch a series by its ID.
+    async def supports_meta(cls, meta: FetchMeta) -> bool:
+        """Check if the class supports a specific meta."""
+        return meta == FetchMeta.TRANSLATIONS
```

Comment:
Enum variants can be compared with `is`, which is considered more idiomatic.
```suggestion
        return meta is FetchMeta.TRANSLATIONS
```